CHANGES_NEXT_RELEASE

```diff
@@ -1,4 +1 @@
-- FIX Update IOTA to the last version of the library.
-- ADD bin section to the package.json to publish commands.
-- ADD Configuration selection in executable.
-- ADD Special configuration for tests.
\ No newline at end of file
+- Implement generic module in the parser (#21).
```

Comment:
Final blank line appreciated... :)

Related to this and just to use the same syntax, aren't we using the prefix to indicate the type of change? :) In the STH, we used a tag in square brackets indicating the type of change: BUG, FEATURE, TASK, etc. -> https://github.com/telefonicaid/IoT-STH/blob/develop/CHANGES_NEXT_RELEASE